pkg/http/api.go

```diff
@@ -205,6 +207,17 @@ func (a *api) constructActorEndpoints() []Endpoint {
 	}
 }
 
+func (a *api) constructTransactionEndpoints() []Endpoint {
+	return []Endpoint{
+		{
+			Methods: []string{http.Post},
+			Route:   "state/transaction",
```

Comment:
Great!, Thanks for making the changes. Since the last review, Multi state store changes have been merged. Now following changes would also be needed:

The url would be state/strorename/transaction.
Also please incorporate the changes to expose it over gRPC>